examples/camera/2d_top_down_camera.rs

```diff
@@ -0,0 +1,153 @@
+//! This example showcases a 2D top-down camera with smooth player tracking.
+//!
+//! ## Controls
+//!
+//! | Key Binding          | Action        |
+//! |:---------------------|:--------------|
+//! | `Z`(azerty), `W`(US) | Move forward  |
+//! | `S`                  | Move backward |
+//! | `Q`(azerty), `A`(US) | Move left     |
+//! | `D`                  | Move right    |
+
+use bevy::core_pipeline::bloom::BloomSettings;
+use bevy::math::{vec2, vec3};
+use bevy::prelude::*;
+use bevy::sprite::{MaterialMesh2dBundle, Mesh2dHandle};
+
+/// Player movement speed factor.
+const PLAYER_SPEED: f32 = 100.;
+
+/// Camera lerp factor.
+const CAM_LERP_FACTOR: f32 = 2.;
+
+#[derive(Component)]
+struct Player;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(ClearColor(Color::BLACK))
+        .add_systems(
+            Startup,
+            (setup_scene, setup_instructions, setup_camera).chain(),
```

Comment:
These systems are not dependent on each other, so there is no need to `chain` them.